commands/omni_drive.py

```diff
@@ -0,0 +1,34 @@
+from wpilib.command import Command, CommandGroup
+
+class OmniDrive(Command):
+
+    def __init__(self, robot, name=None, timeout=None):
+        """This is the constructor of the command, use this to declare subsystem dependencies"""
+        #Use self.requires() here to declare subsystem dependencies
+        #eg. self.requires(chassis)
+        super().__init__(name, timeout)
+        self.requires(robot.chassis)
+        self.robot = robot
+
+    def initialize(self):
+        """Called just before this Command runs the first time"""
+        robot.drive(0.0, 0.0, 0.0, 0.0) # just to be safe
```

Comment:
It would be better if we called the safe position by using a variable from the robot map.